events/hfe_events_plague.txt

```diff
@@ -1938,8 +1945,11 @@ country_event = {
 	option = {
 		name = hfe_plague.58.0
 		trigger = {
-			not = {
-				has_tech_option = hfe_tech_plague_immune
+			OR = {
```

Comment:
I think it should be nand

Reply:
NOT-AND = NAND

Reply:
And you have one or and a lot of nots instead of one nand.

Reply:
I'll die before I use NAND.